gitlab-runner/src/lib.rs

```diff
@@ -60,7 +60,7 @@ pub use client::Phase;
 /// `get_path` method is required so that the globbing Gitlab expects
 /// can be performed without the handler needing to be involved.
 #[async_trait::async_trait]
-pub trait UploadableFile: Eq {
+pub trait UploadableFile {
```

Comment:
This is a semver break; Also could we have a commit message that's a bit more then "it is annoying" -- I'm not sure why you're changing this tbh  :)